src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { TodosContext } from '../../TodosContext';
+import { Todo } from '../../types/todo';
+
+type Props = {};
+
+export const TodoApp: React.FC<Props> = () => {
+  const { setTodos } = useContext(TodosContext);
```

Comment:
Create a hook for your context instead of passing `TodosContext` every time

`const useTodos = useContext(TodosContext);`